andela/activo-api
# #158592755 Filter Query by Deleted field

#### What does this PR do?
>- Filters deleted records in the `filter_query` method
#### Description of Task to be completed?
>- Filter deleted records in the `filter_query` method
#### How should this be manually tested?
>- `git fetch origin ch-query-impl-158592755:ch-query-impl-158592755`, then `git checkout ch-query-impl-158592755`
>- Start the application `python manage.py`
>- Delete an AssetCategory
>- Send a GET request to this endpoint: `localhost:5000/api/v1/asset-categories`
>- Then deleted record should not be returned
#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- [#158592755](https://www.pivotaltracker.com/story/show/158592755)
#### Screenshots (if appropriate)
>- N/A
#### Questions:
>- N/A
